### ISSUE TITLE
Project generated by angel init with template graphql does not work out of the box

### ISSUE THREAD

**Issue by InterFelix** [dukefirehawk/angel#2]
When I try to run the application right after initializing the project, the server starts as expected.
However, when I open GraphiQL, I can see the interface, but it immediately throws an error.
The stacktrace in the dart logs does not include any mention of a file within the project, only the packages.

Here's the stacktrace in case it is of any use to you:
```
[SEVERE] scoutventory_backend: An error occurred while processing GraphQL query at /graphql.

type 'Null' is not a subtype of type 'Map<dynamic, dynamic>' in type cast
#0      graphQLHttp.<anonymous closure>.executeMap (package:angel3_graphql/src/graphql_http.dart:63:55)
<asynchronous suspension>
#1      graphQLHttp.<anonymous closure> (package:angel3_graphql/src/graphql_http.dart:83:20)
<asynchronous suspension>
#2      Driver.runPipeline (package:angel3_framework/src/core/driver.dart:406:22)
<asynchronous suspension>
#3      HotReloader.handleRequest (package:angel3_hot/angel3_hot.dart:119:14)
<asynchronous suspension>
```

**Comment by dukefirehawk** [comment]
This issue has been resolved in angel3_graphql 2.0.2 release. Run "pub update" should update all the dependencies.

**Comment by InterFelix** [comment]
Can confirm.
Closing this issue now.
